src/components/AddBookForm/AddBookForm.js

```diff
@@ -6,16 +6,19 @@ const AddBookForm = () => {
   const [title, setTitle] = useState();
   const [author, setAuthor] = useState();
   const dispatch = useDispatch();
+
   const addBookHandeler = (event) => {
     event.preventDefault();
     if (!title || !author) return;
     const book = {
       title,
       author,
-      id: Math.floor(Math.random() * 10000).toString(),
+      item_id: Math.floor(Math.random() * 10000).toString(),
+      category: 'Fiction',
     };
     dispatch(addBook(book));
   };
```

Comment:
- [ ] (Optional) In your `addBookHandeler` function, I would advise you to set the state of the title and author to an empty string after you submit a book. This would make the UI friendly and a user won't always have to clear the input field every time they want to add a new book. 👍